### PR TITLE
Remove merge fx macro

### PR DESCRIPTION
**Pull request by yenda**
### Summary:

Replace all usage of `handlers-macro/merge-fx` by `fx/merge`

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/events.cljs [hunk 1]
```diff
@@ -103,13 +103,17 @@
...
   (let [app-coming-from-background? (= state "active")]
-    (handlers-macro/merge-fx cofx
-                             {::app-state-change-fx state
-                              :db                   (assoc db :app-state state)}
-                             (inbox/request-messages app-coming-from-background?)
-                             (hardwallet/return-back-from-nfc-settings app-coming-from-background?))))
+    (fx/merge cofx
+              {::app-state-change-fx state
+               :db                   (assoc db :app-state state)}
+              (on-return-from-background))))
```
Comment (dmitryn) [inline src/status_im/ui/screens/events.cljs]: `app-coming-from-background?` is not used, is it expected?

File: src/status_im/init/core.cljs [hunk 2]
```diff
@@ -168,38 +170,46 @@
...
+  (fx/merge cofx
+            {:web3/set-default-account    [web3 address]
+             :web3/fetch-node-version     [web3
+                                           #(re-frame/dispatch
+                                             [:web3/fetch-node-version-callback %])]
+             :notifications/get-fcm-token nil}
+            (initialize-account-db address)
+            (protocol/initialize-protocol address)
+            (models.contacts/load-contacts)
+            #(when (dev-mode? %)
```
Comment (dmitryn) [inline src/status_im/init/core.cljs:202]: does it need to be wrapped into anonymous fn?

Comment (dmitryn) [reply src/status_im/init/core.cljs:202]: nevermind, i get the idea now, `%` parameter is a cofx

**Comment by lukaszfryc** [comment]
@yenda I spend some time testing it and the only bug I noticed is that if I click "Add to contacts" button in 1-1 chat this error appears:

**Review by dmitryn — APPROVED** [review]

File: src/status_im/ui/screens/events.cljs (hunk 1 above)
Comment (yenda) [reply src/status_im/ui/screens/events.cljs]: thanks good catch

**Review by janherich — APPROVED** [review]

**Review by rasom — APPROVED** [review]

**Review by flexsurfer — APPROVED** [review]

**Pull request merged**